cat-theme/src/templates/footer/footer.ftl

```diff
@@ -0,0 +1,16 @@
+<footer class="footer" id="footer">
+    <div class="footer--top">
+        <a class="footer--phone">Phone Number Here</a>
+        <a class="footer--email">Email Here</a>
+    </div>
+
+    <div class="footer--bottom">
+        <div class="copyright">@ 2019 Cat Company</div>
+
+        <ul>
+            <li>
+                <a>Facebook</a>
```

Comment:
There is no href link for Facebook.  Since this is in the footer, I would assume this would be a theme setting.